packages/react-accessible-tooltip/src/Tooltip.test.js

```diff
@@ -53,52 +57,112 @@ function testReact(React, Tooltip) {
         closeButton = wrapper.find(CloseButton);
     });
 
-    it(`${React.version} - asdmatches the previous snapshot`, () => {
```

Comment:
This stuff is pretty much all just indentation changes, as I wrapped them in an extra `describe` declaration.